xc7/tests/ddr/arty.sdc

```diff
@@ -0,0 +1,13 @@
+create_clock -period 5 sys4x_clk__main_clkout_buf1 -waveform {0.000 2.500}
```

Comment:
Some of these clocks are related?  Do they need a clock group or some such?  I'm not totally clear.  @mithro ?